lib/assert.js

```diff
@@ -38,34 +45,33 @@ const assert = module.exports = ok;
 //                             actual: actual,
 //                             expected: expected });
 
-assert.AssertionError = function AssertionError(options) {
-  this.name = 'AssertionError';
-  this.actual = options.actual;
-  this.expected = options.expected;
-  this.operator = options.operator;
-  if (options.message) {
-    this.message = options.message;
-    this.generatedMessage = false;
-  } else {
-    this.message = getMessage(this);
-    this.generatedMessage = true;
-  }
-  var stackStartFunction = options.stackStartFunction || fail;
-  Error.captureStackTrace(this, stackStartFunction);
-};
-
-// assert.AssertionError instanceof Error
-util.inherits(assert.AssertionError, Error);
-
-function truncate(s, n) {
-  return s.slice(0, n);
+// TODO(jasnell): Consider moving AssertionError into internal/errors.js
```

Comment:
I expected at least one person to ask about this one :-) ... if we think it's a good idea, then I can just add a commit to this PR that moves it.  It would be a minimally invasive change and would allow us to simplify some of the code. I just didn't do it yet because I want to know if there are any objections before doing so.